items.md

```diff
@@ -143,3 +207,21 @@ It returns the collection where the item belong to
 Example: to be provided
 
 It returns the collection that have the item as template
+
+### Relationships per item
+**/api/core/items/<:uuid>/relationships**
+
+A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/items/5a3f7c7a-d3df-419c-b8a2-f00ede62c60a/relationships
+
+It embeds all relationships where either the left or the right item matches the given uuid
+
+## Deleting a collection
```

Comment:
Should say "Deleting an item"

Reply:
This is actually not modified in this PR, just merged from master. The error is in https://github.com/DSpace/Rest7Contract/blob/master/items.md#deleting-a-collection
Shouldn't we rather update it outside this PR?

Reply:
Ok, I missed that. Yes, we can update this outside of this PR.  Resolving this conversation.